### PR TITLE
Add skeleton UI

### PR DESCRIPTION
**Pull request by tjingsheng**

### CODE REVIEW

**Comment by tjingsheng** [comment]
DO NOT APPROVE YET

File: src/main/java/seedu/address/seplendidui/ItemDetailPanel.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+     */
+    public void setItemDetail(SeplendidDataType item) {
+        // Clear the existing items and add the selected item
+        itemDetailView.getItems().clear();
+        if (item != null) {
+            itemDetailView.getItems().add(item);
+        }
+    }
+
+    //TODO: Convert it to single item instead of list, detail panel should always be a single item.
```
Comment (lamchenghou) [inline src/main/java/seedu/address/seplendidui/ItemDetailPanel.java:43]: Yep, agreed.

File: src/main/java/seedu/address/seplendidui/MainWindow.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+        itemListPanel = new ItemListPanel<>(itemList, itemDetailPanel);
+        itemListPanelPlaceholder.getChildren().add(itemListPanel.getRoot());
+
+        resultBox = new ResultBox();
+        resultBoxPlaceholder.getChildren().add(resultBox.getRoot());
+
+        CommandBox commandBox = new CommandBox(this::executeCommand);
+        commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
+    }
```
Comment (lamchenghou) [inline src/main/java/seedu/address/seplendidui/MainWindow.java:90]: Note: TODO: Need to add support, and switching between different `ObservableList`s for different commands.

**Review by lamchenghou — APPROVED** [review]
Great work! I'll merge first to delivery v1.2.

**Pull request merged**